grails-app/services/au/org/ala/profile/hub/ProfileService.groovy

```diff
@@ -320,11 +322,57 @@ class ProfileService {
             case ReportType.DRAFT_PROFILE:
                 resp = webService.get("${grailsApplication.config.profile.service.url}/report/draftProfiles?opusId=${enc(opusId)}")
                 break;
+            case ReportType.MOST_RECENT_CHANGE:
+                dates.period = dates.period?:'last30Days';
```

Comment:
Use the value from DateRangeType.LAST_30DAYS instead of hardcoding the string here